simple-image-load-transform.md

```diff
@@ -93,29 +93,61 @@ You can chain transformations as shown below, write your own classes that will a
 ImageTransform transform = new MultiImageTransform(randNumGen, new CropImageTransform(10), new FlipImageTransform(),new ScaleImageTransform(10), new WarpImageTransform(10));
 ~~~
 
+* Advanced: Pipeline image transform
+
+If you want to create a more sophisticated pipeline, you can use `PipelineImageTransform`.
+You can create image transform pipelines, either sequentially or randomly.
+
+In order to employ this complex pipeline, you can create a list,`List<Pair<ImageTransform, Double>>`, 
+then add ImageTransforms which you want to apply to the raw images with double values which is a probability.
+Note : the probability represents particular element in the pipeline will be executed. 
+This is helpful for creating complex pipelines. 
+
+Here is the code snippet of PipelineImageTransform.
+```java
+boolean shuffle = false;
+ImageTransform randCrop = new RandomCropTransform(160,160);
+ImageTransform flip = new FlipImageTransform();
+
+List<Pair<ImageTransform, Double>> pipeline = new LinkedList<>();
+pipeline.add(new Pair<>(randCrop, 1.0));
+pipeline.add(new Pair<>(flip, 0.5));
+
+ImageTransform transform = new PipelineImageTransform(pipeline, shuffle);
+```
+The pipeline can also be randomly shuffled with each transform in order to increase the available dataset.  
+If you set `shuffle` as true, you can get a image pipelines that randomly execute.
```

Comment:
If you set `shuffle` to true, you can get an image pipeline that executes each transformation at random.